Resources/doc/installation.md

```diff
@@ -0,0 +1,82 @@
+# Installation
+
+You can install this bundle with `composer` using the `sulu/validation-bundle` package.
+
+```bash
+composer require sulu/validation-bundle
+```
+
+## 1. Add SuluValidation to your project:
+
+In `composer.json`:
+
+```
+"sulu/validation-bundle": "~0.1"
```

Comment:
As this bundle is under heavy development people should probably use the latest version of it. So here the latest version should be used. What do you think?